### PR TITLE
Add place summary json as gke config for production config update

### PR DESCRIPTION
**Pull request by shifucun**
This PR has two goals:

1. Update place summary json without restarting the server. This is done by using GKE config map mounted as a disk volume. The update is done by running `update_place_summary_config.sh` which updates the config map.
2. Version control and able to review the place summary json change. The checked-in summary json file is used as the golden file.

There are two scenarios:

- In regular deployment, the server restarts with the checked in summary json file.
- For ad-hoc summary update. Make the change in codebase, send for code review and check in. Then run `update_place_summary_config.sh` to push the change to GKE config. The push can also be configured in a cloudbuild to automate the process.

### CODE REVIEW

File: server/routes/place/html.py [hunk 1]
```diff
@@ -31,6 +30,17 @@
     "Climate": "Environment",
 }
 
+PLACE_SUMMARY_FILE_PATH = "/datacommons/place-summary/place_summary_content.json"
+
+
+def get_place_summaries() -> dict:
+  """Load place summary content from disk"""
+  if os.path.isfile(PLACE_SUMMARY_FILE_PATH):
+    with open(PLACE_SUMMARY_FILE_PATH) as f:
```
[the remark this answers is not in the document]
Comment (shifucun) [reply server/routes/place/html.py:39]: Done

File: server/routes/place/html.py [hunk 2]
```diff
@@ -84,33 +94,28 @@ def place(place_dcid=None):
...
-                               maps_api_key=current_app.config['MAPS_API_KEY'])
+    if os.environ.get('FLASK_ENV') in [
+        'local', 'autopush', 'dev', 'staging', 'production'
+    ]:
+      # Fetch summary text from GCS bucket and log timing
+      start_time = time.time()
+      place_summary = get_place_summaries().get(place_dcid, {})
+      elapsed_time = (time.time() - start_time) * 1000
+      logging.info(
+          f"Place page summary fetch from GCS took {elapsed_time:.2f} milliseconds."
```
[the remark this answers is not in the document]
Comment (shifucun) [reply server/routes/place/html.py:108]: Removed the timing as it is not needed anymore

[the remark this answers is not in the document]
Comment (pradh) [reply server/routes/place/html.py]: Maybe keep it for a bit to confirm latency?  First access may hit disk (which should be probably under 10ms), and subsequent local file should be cached in memory by filesys, so might be even faster

[the remark this answers is not in the document]
Comment (shifucun) [reply server/routes/place/html.py]: Done

File: scripts/deploy_gke_helm.sh [hunk 3]
```diff
@@ -147,7 +147,8 @@ function deploy_website() {
   --set cronTesting.webApiRoot="$WEBSITE_SERVICE_URL" \
   --set cronTesting.nodejsApiRoot="$NODEJS_SERVICE_URL" \
   --set-file nl.embeddings=deploy/nl/embeddings.yaml \
-  --set-file nl.models=deploy/nl/models.yaml
+  --set-file nl.models=deploy/nl/models.yaml \
+  --set-file website.placeSummary=server/place_summary_content.json
```
Comment (juliawu) [inline scripts/deploy_gke_helm.sh:151]: let's use the `server/config/summaries/` folder instead.

Comment (shifucun) [reply scripts/deploy_gke_helm.sh:151]: Done

**Pull request merged**